core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -181,6 +185,13 @@ describe('Topic editor functionality', function() {
       await topicsAndSkillsDashboardPage.assignSkillWithIndexToTopicByTopicName(
         0, TOPIC_NAME);
 
+      await topicsAndSkillsDashboardPage.get();
+      await topicsAndSkillsDashboardPage.navigateToSkillsTab();
+      await topicsAndSkillsDashboardPage.filterSkillsByStatus(
+        Constants.SKILL_STATUS_UNASSIGNED);
+      await topicsAndSkillsDashboardPage.assignSkillWithIndexToTopicByTopicName(
+        0, TOPIC_NAME);
```

Comment:
Can we also test deleting this skill, since that will no longer be tested by topicsAndSkillsDashboard?

Reply:
Ah, Hey @U8NWXD sorryf for the confusion, I don't think we need this here, I removed becuase it was a redundant test. We are still testing deleting the skill in the topicsAndSkillDashboard(see [here](https://github.com/oppia/oppia/blob/f25ee87d23300cac45b09f77dd2f2872fba55d8f/core/tests/protractor_desktop/topicsAndSkillsDashboard.js#L56).)